### ISSUE TITLE
Screen reader support (eg: JAWS, etc)

### ISSUE THREAD

**Issue by litui** [keybase/keybase-issues#207]
For accessibility, some guidelines should be followed to ensure screen-reader clarity and ease of navigation.  This ties in #14.

Tools such as [Fangs](https://addons.mozilla.org/en-us/firefox/addon/fangs-screen-reader-emulator/) can help with ensuring screen-reader users will be able to access the site effectively.

[Browsing habits of screen reader users](http://www.standards-schmandards.com/2005/browsing-habits/) provides some (possibly a little dated since the article is from 2005) detail on what the implications are in terms of markup.
